src/main/java/com/blockout/views/ScrollingView.java

```diff
@@ -52,11 +42,23 @@ private void setup()
         scrollbar.putInside(this);
     }
 
+    @NotNull
     protected ScrollingContainer createScrollingContainer()
     {
         return new ScrollingContainer(this);
     }
 
+    /**
+     * Load from xml.
+     *
+     * @param params xml parameters.
+     */
+    public ScrollingView(PaneParams params)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this constructor to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1213)